package.json

```diff
@@ -119,9 +119,9 @@
     "@babel/preset-env": "^7.10.4",
     "@babel/preset-typescript": "^7.13.0",
     "@babel/register": "^7.8.6",
-    "@onfido/castor": "^1.2.0",
+    "@onfido/castor": "1.1.2",
```

Comment:
Interesting, why downgrade?

Reply:
As discussed on Slack, it was downgraded due to a build error after upgrading. But now fixed after upgrading `sass` to v1.33.0 as suggested. Thanks! 😃